docs/src/opt_cep.md

```diff
@@ -84,6 +105,15 @@ using Clp
 optimizer=Clp.Optimizer
 ```
 
+## Solver Configuration
+Depending on the Solver different solver configurations are possible. The information is always provided as `Dict{Symbol,Any}`. The keys of the dictionary are the parameters and the values of the dictionary are the values passed to the solver.
+
+For example the `Gurobi` solver can be configured to have no OutputFlag and run on two threads (per julia thread) the following way:
+```julia
+optimizer_config=Dict{Symbol,Any}(:OutputFlag => 0, :Threads => 2)
```

Comment:
Just out of curiosity: Have you found that it becomes faster if you run it on multiple threads (but still one core)? Or did you also increase the number of cores?